tests/unit/s2n_crl_test.c

```diff
@@ -0,0 +1,88 @@
+/*
+* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+#include "s2n_test.h"
+#include "testlib/s2n_testlib.h"
+#include "tls/s2n_crl.h"
+
+DEFINE_POINTER_CLEANUP_FUNC(struct s2n_x509_crl*, s2n_x509_crl_free);
+
+int free_uint8_array_pointer(uint8_t** array) {
+    if (array && *array) {
+        free(*array);
+    }
+    return S2N_SUCCESS;
+}
+
+#define S2N_CRL_ROOT_CERT                               "../pems/crl/root_cert.pem"
```

Comment:
#defines for the whole file usually appear above all methods so that you don't have to reason about when they're available or not. So it's a little odd that these appear below free_uint8_array_pointer.

Reply:
Right okay. I moved this to the top of the file.